modules/monitoring/application/controllers/HostsController.php

```diff
@@ -151,6 +153,16 @@ public function showAction()
         $this->view->commentsLink = Url::fromRequest()->setPath('monitoring/list/comments');
         $this->view->sendCustomNotificationLink = Url::fromRequest()
             ->setPath('monitoring/hosts/send-custom-notification');
+
+        $this->view->extensionsHtml = array();
+        foreach (Hook::all('Monitoring\DetailviewExtension') as $hook) {
+            /** @var DetailviewExtensionHook $hook */
+            $module = $this->view->escape($hook->getModule()->getName());
+            $this->view->extensionsHtml[] =
```

Comment:
The extension should only be added to the array if the hook does not return the empty string. Else we may render empty containers.